test/integration/targets/portmake/tasks/portmake.yml

```diff
@@ -0,0 +1,264 @@
+####
+# setup portstree
+
+- name: silent install of subversion without module dependency
+  shell: pkg install -y subversion >/dev/null 2>&1
+
+- name: silently checkout a tested revision of the ports tree to make the tests more stable
+  shell: svn checkout -r 493256 https://svn.FreeBSD.org/ports/head /usr/ports >/dev/null 2>&1
+
+###
+# check ports setup
+- name: check for existence of screen directory
+  stat:
+    path: /usr/ports/sysutils/screen
+  register: dir_sysutils_screen
+  
+- fail:
+    msg: "could not find directory /usr/ports/sysutils/screen"
+  when: dir_sysutils_screen.stat.exists == False
+
+- name: check for existence of sudo directory
+  stat:
+    path: /usr/ports/sysutils/screen
+  register: dir_security_sudo
+  
+- fail:
+    msg: "could not find directory /usr/ports/security/sudo"
+  when: dir_security_sudo.stat.exists == False
+
+####
+# test manual compilation and installation of packages
+
+- name: pkg remove screen
+  shell: pkg remove screen
+  args:
+    deletes: /usr/local/bin/screen  
+  ignore_errors: true
+  
+- name: pkg remove sudo
+  shell: pkg remove sudo
+  args:
+    deletes: /usr/local/bin/sudo
+  ignore_errors: true
+
+- name: test make install screen
+  shell: make install clean
```

Comment:
This task is creating excessive (over 16 MB) output according the Shippable logs I retrieved:

```
06:38 TASK [portmake : test make install screen] *************************************
console_limit_errorConsole size exceeds 16 MB limit. Truncated from here.
```

The task should output a few KB at most, not multiple MB.

Reply:
Since the truncation is reported here, and the logs prior to this point are approximately 170 KB I'm guessing the output is one very long line.

Reply:
Since the output of the shell command will include stdout/stdout_lines and stderr/stderr_lines, the output of `make install clean` would only need to reach ~8 MB to result in over 16 MB of output for the task.